src/test/java/com/assignmentTest/Assignment2Test.java

```diff
@@ -0,0 +1,23 @@
+package com.assignmentTest;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import com.assignment.Assignment2;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import sun.jvm.hotspot.utilities.Assert;
```

Comment:
There is a compile time error here. Can you please make sure that there is no compile time error before you push your code?
@shrikanttechincal @PranavHanwante  Please make sure you check this